config.json

```
@@ -295,6 +295,22 @@
                 "MAX_LIMIT": 1,
                 "WINDOW_SIZE": 1440
             }
+        },
+        {
+            "ID": "Monsoon",
+            "NAME": "Monsoon testnet",
+            "TOKEN": "RDP",
+            "RPC": "https://testnet.monsoon.rainfall.one/ext/bc/rDJjduekrs96YivEy2i4fHLM8Xt4sEWZn5gSFqoRhZe1fK2Hu/rpc",
+            "CHAINID": 100610,
+            "EXPLORER": "http://scout.testnet.monsoon.rainfall.one/",
```

Let's switch to `https`?
```suggestion
            "EXPLORER": "https://scout.testnet.monsoon.rainfall.one",
```